contributing/content-style-guide.md

```diff
@@ -529,6 +530,8 @@ dbt Cloud CLI | CLI, dbt CLI
 dbt Core | CLI, dbt CLI
 </div></b>
 
+Make sure you keep the reader close to the content and documentation (second person). For example, to explain that a feature is available to a particular dbt Cloud plan &mdash; use “XYZ is available on Enterprise plans” or “People on Enterprise plans can XYZ..” to keep users closer to the documentation. This will signal to users to check their plan or account status on their own. 
```

Comment:
can change to this from thread and bypass docs sync sugg

```suggestion
Make sure you keep the reader close to the content and documentation (second person). For example, to explain that a feature is available to a particular dbt Cloud plan &mdash; use “XYZ is available on Enterprise plans” or “Users on Enterprise plans can XYZ..” to keep users closer to the documentation. This will signal to users to check their plan or account status on their own. 


```